Sources/StreamChatUI/UIConfig+MessageComposer.swift

```diff
@@ -43,8 +43,8 @@ public extension _UIConfig {
         public var composerButton: UIButton.Type = UIButton.self
         public var textView: _ChatMessageComposerInputTextView<ExtraData>.Type = _ChatMessageComposerInputTextView<ExtraData>.self
 
-        public var checkmarkControl: _ChatMessageComposerCheckmarkControl<ExtraData>.Type =
-            _ChatMessageComposerCheckmarkControl<ExtraData>.self
+        public var checkmarkControl: _ChatCheckmarkControl<ExtraData>.Type =
+            _ChatCheckmarkControl<ExtraData>.self
```

Comment:
This can also go to the root of the config. But, this component is probably not even needed, since it is so simple, but on the refactor of composer view that will be decided, so no need to change anything here.